app-office/kapow/kapow-1.5.10.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="A Qt punch clock program"
+DESCRIPTION="Punch clock program designed to easily keep track your hours.\
+It's written in Qt."
+COPYRIGHT="2018-2019 Grameme gott"
+HOMEPAGE="https://github.com/gottcode/kapow"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/gottcode/kapow/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="21f4c1150f44cedd0f74c8b5ad795e90ca636ee90faae22149e542c75c92900b"
+SOURCE_FILENAME="kapow-$portVersion.tar.gz"
+SOURCE_DIR="kapow-$portVersion"
+ADDITIONAL_FILES="kapow.rdef.in"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+ SECONDARY_ARCHITECTURES="x86"
+fi
+
+PROVIDES="
+	kapow$secondaryArchSuffix = $portVersion
+	app:Kapow = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5PrintSupport$secondaryArchSuffix
+	lib:libQt5Svg$secondaryArchSuffix
+	lib:libQt5Xml$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5PrintSupport$secondaryArchSuffix
+	devel:libQt5Svg$secondaryArchSuffix
+	devel:libQt5Xml$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:lrelease$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:qmake$secondaryArchSuffix >= 5
+	cmd:unzip
+	"
+
+BUILD()
+{
+	qmake
+	make $jobArgs
+}
+
+INSTALL()
+{
+	install -d "$appsDir"
+	install -T ./kapow "$appsDir"/Kapow
+	cp -rf translations "$appsDir"/
```

Comment:
Probably best to install these in $appsDir/Kapow (we don't want $appsDir/translations in there, but it would be OK to have it in $appsDir/Kapow/translations)
PS, no need for the qoutes just appsDir not "$appsDir"

Reply:
.. ok done, update recipe